src/cmdc_tools/datasets/official/NV/data.py

```diff
@@ -0,0 +1,287 @@
+            # Get next page button
+            button = await _get_next_page_button(page)
+            await button.click()
+            # Wait for dashboard to load (2)
+            await page.waitForXPath("//div[text()='COVID-19 Statistics by County']")
+            # Go to next page
+            await button.click()
+            # Wait for dashboard to laod (3)
+            await page.waitForXPath("//div[text()='Results Filter for Demographics']")
+            # Go to next page
+            await button.click()
+            # Wait for dashboard to laod (3)
+            await page.waitForXPath("//div[text()='Cumulative Tests Reported']")
+            # Go to next page
+            await button.click()
+            # Wait for dashboard to laod (3)
+            await page.waitForXPath("//div[text()='Cumulative Cases']")
+            # Go to next page
+            await button.click()
+            # Wait for dashboard to laod (3)
+            await page.waitForXPath("//div[text()='Cumulative Deaths']")
+            # Go to next page
+            await button.click()
+            # Wait for dashboard to laod (3)
+            await page.waitForXPath(
+                "//div[text()='Daily Growth Rate, Cases (April 1st to Current)']"
+            )
+            # Go to next page
+            await button.click()
+            # Find graph
+            graph = await page.waitForXPath("//*[@class='cartesianChart']")
+            # return graph
+            # all_labels = []
+            # labels = await self._get_labels_from_single_graph(page, graph[0])
+            # icu_labels = await self._get_labels_from_single_graph(page, graph[1])
+
+            await graph.click(button="right")
+            # Get table button
+            table_button = await page.waitForXPath("//h6[text()='Show as a table']")
+            await table_button.click()
+
+            labels = await self._get_labels_from_graph(page)
+            # parse labels
+            data = {"Suspected": [], "Confirmed": []}
+            for label in labels:
+                split = label.split(". ")
+                date = split[0].split("Date")[1].strip()
+                tests = split[1].split(" ")
+                tests_type = tests[0].strip()
+                # Skip all new tests
+
+                tests_num = int(tests[1][:-1].strip().replace(",", ""))
+                {"Date": date, f"{tests_type}": tests_num}
+
+                # data.append({"Date": date, f"{tests_type}": tests_num})
+                data[tests_type].append({"dt": date, "value": tests_num})
+            suspected = pd.DataFrame(data["Suspected"]).assign(
+                variable_name="hospital_beds_in_use_covid_suspected"
+            )
+            confirmed = pd.DataFrame(data["Confirmed"]).assign(
+                variable_name="hospital_beds_in_use_covid_confirmed"
+            )
+
+            df = pd.concat([suspected, confirmed], sort=False, ignore_index=True)
```

Comment:
If we're collecting `suspected` and `confirmed` then we should also make sure that we collect `total`.